src/Bundle/BundleMetadata.php

```diff
@@ -43,14 +43,14 @@ class BundleMetadata
     protected $name;
 
     /**
-     * @var bool
+     * @var string
      */
-    protected $extendedDirectory = false;
+    protected $extendedDirectory;
```

Comment:
this looks like a BC break

Reply:
No BC break, because we set it every time we create a new instance:
https://github.com/sonata-project/SonataEasyExtendsBundle/blob/1954ef9aa2a9aa046d02a914c911bda09120fba7/src/Bundle/BundleMetadata.php#L234-L237

Reply:
The getter also returns a string:
https://github.com/sonata-project/SonataEasyExtendsBundle/blob/1954ef9aa2a9aa046d02a914c911bda09120fba7/src/Bundle/BundleMetadata.php#L120-L126